pkg/validation/validation.go

```diff
@@ -55,3 +59,22 @@ func ValidatePassword(i interface{}, k string) (s []string, errs []error) {
 
 	return
 }
+
+// ValidatePrivilege validates the privilege is in the authorized set.
+// Will also check for the ALL privilege and hopefully provide a helpful error message.
+func ValidatePrivilege(valid []string, ignoreCase bool) schema.SchemaValidateFunc {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1019: schema.SchemaValidateFunc is deprecated: please use SchemaValidateDiagFunc  (staticcheck)